domain/src/main/java/camp/nextstep/edu/github/domain/GithubRepository.kt

```diff
@@ -0,0 +1,5 @@
+package camp.nextstep.edu.github.domain
+
+interface GithubRepository {
+    suspend fun getRepositories(): List<RepositoryVO>?
```

Comment:
```suggestion
    suspend fun getRepositories(): List<RepositoryVO>
```
컬렉션은 null일 경우 빈 리스트를 반환하도록 하는 것을 권장해요.